applications/box.md

```diff
@@ -0,0 +1,213 @@
+
+Below we provide an **example roadmap**. In the descriptions, it should be clear how your project is related to IPFS/Crust. We *recommend* that the scope of the work can fit within a three-month period and that teams structure their roadmap as 1 milestone ≈ 1 month. 
+
+For each milestone,
+
+* make sure to include a specification of your software. _Treat it as a contract_; the level of detail must be enough to later verify that the software meets the specification.
+* include the amount of funding requested _per milestone_.
+* include documentation (tutorials, API specifications, architecture diagrams, whatever is appropriate) in each milestone. This ensures that the code can be widely used by the community.
+* provide a test suite, comprising unit and integration tests, along with a guide on how to set up and run them.
+* commit to providing Dockerfiles for the delivery of your project. 
+* indicate milestone duration as well as number of full-time employees working on each milestone, and include the approximate number of days along with the cost per day.
+* _Deliverables 0a-0d are mandatory_ and shall not be removed, unless you explicitly specify a reason within the PR's `Additional Notes` section (e.g. Milestone X is research oriented and as such there is no code to test).
+
+> If any of your deliverables is based on somebody else's work, make sure you work and publish _under the terms of the license_ of the respective project and that you **highlight this fact in your milestone documentation** and in the source code if applicable! **Teams that submit others' work without attributing it will be immediately terminated.**
+
+### Overview
+* **Total Estimated Duration:** 3 months
+* **Full-Time Equivalent (FTE):**  3 months as we are working on it full-time
+* **Total Costs:** $28K
+
+### Milestone 1 — Implement Crust Order Modules 
+* **Estimated Duration:** 1 month
+* **FTE:**  2
+* **Costs:** 10,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | Apache 2.0 / MIT / Unlicense |
+| 0b. | Documentation | We will provide both inline documentation of the code and a basic tutorial that explains how a user can (for example) spin up one of our Crust nodes. Once the node is up, it will be possible to send test transactions that will show how the new functionality works. |
+| 0c. | Testing Guide | Core functions will be fully covered by unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. | 
+| 1. | File transfer module from applications to box | We will create an IPFS+Libp2p module that transfers the file from application to the box backend |  
```

Comment:
This is updated in response to: https://github.com/crustio/Crust-Grants-Program/pull/39/files#r669246898

```suggestion
| 1. | File transfer module from applications to box | We will create an IPFS+Libp2p module that transfers the file from application to the box backend. User does not need to run IPFS on the end device(e.g. mobile). box runs the ipfs node and provides APIs that talks to front-end app on the device through libp2p. This is transparent from end-user's perspective. |  
```